libs/lib_parsing/Origin.mli

```diff
@@ -0,0 +1,77 @@
+(* Cooper Pierce
+ *
+ * Copyright (c) Semgrep Inc.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation, with the
+ * special exception on linking described in file LICENSE.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
+ * LICENSE for more details.
+ *)
+
+(** Origins of content which may be consumed by Semgrep.
+
+   A source is the {e user-relevant origin} of content (e.g., source code).
```

Comment:
Need to update all the comment to not use "source" anymore but "origin".

Reply:
Thought I got them all 😔

Reply:
Done in 288cddf0d3fc44b07b1539f997a425128dceaeb6